.circleci/config.yml

```diff
@@ -193,8 +193,15 @@ jobs:
       - run:
           name: 'Unit Tests'
           command: node build-system/pr-check/unit-tests.js
+          environment:
+            JUNIT_REPORT_PATH: result-reports/
```

Comment:
Would it be possible to set this environment variable in `runtime-test-base.js` right when you're adding the reporter to the config? That way, the code will be automatically applicable to all the testing jobs without having to update half a dozen places in this config file.

Reply:
Interestingly despite the documentation mentioning this step this appears to not be required when `store_test_artifacts` is set so I'll just remove this.
https://app.circleci.com/pipelines/github/ampproject/amphtml/6296/workflows/824a7fd0-be4b-48a1-815c-c779c1c8584c/jobs/89674/steps